src/diffusers/utils/hub_utils.py

```diff
@@ -144,6 +144,36 @@ def create_model_card(args, model_name):
     model_card.save(card_path)
 
 
+# Taken from `transformers`
+def create_and_tag_model_card(repo_id: str, token: Optional[str] = None):
+    """
+    Creates or loads an existing model card and tags it with the `library_name`.
+
+    Args:
+        repo_id (`str`):
+            The repo_id where to look for the model card.
+        token (`str`, *optional*):
+            Authentication token, obtained with `huggingface_hub.HfApi.login` method. Will default to the stored token.
+    """
+    if not is_jinja_available():
+        raise ValueError(
+            "Modelcard rendering is based on Jinja templates."
+            " Please make sure to have `jinja` installed before using `create_and_tag_model_card`."
+            " To install it, please run `pip install Jinja2`."
+        )
+
```

Comment:
```suggestion
```

Throwing a good error message is already handled by the `huggingface_hub` library - no need to do it again here